### ISSUE TITLE
Tree view is empty

### ISSUE THREAD

**Issue by Emaro** [dendronhq/dendron#263]
**Describe the bug**

The dendron tree view displays no items, although there are plenty of notes.

**To Reproduce**

1. Open Dendron workspace.

Restarting VS Code and dis- and reenabling the Dendron extension didn't help.

**Expected behavior**

The tree view displayes the note hierarchy.

**Screenshots**

![dendron empty tree view screenshot](https://user-images.githubusercontent.com/25137789/95653091-2db4fa80-0af6-11eb-98b4-9d884665767d.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - VS Code 1.50
 - Version: 0.12.6

**Dendron Log file**

```
{"level":30,"time":1602402995665,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"_activate","stage":"dev","isDebug":false,"logPath":"b:\\AppData\\Roaming\\Code\\logs\\20201011T095444\\exthost1\\dendron.dendron","extensionPath":"c:\\Users\\User\\.vscode\\extensions\\dendron.dendron-0.12.6","extensionUri":{"$mid":1,"fsPath":"c:\\Users\\User\\.vscode\\extensions\\dendron.dendron-0.12.6","_sep":1,"path":"/c:/Users/User/.vscode/extensions/dendron.dendron-0.12.6","scheme":"file"},"storagePath":"b:\\AppData\\Roaming\\Code\\User\\workspaceStorage\\0d7fa9d2843e7172eb2840b215215ea2\\dendron.dendron"}
{"level":30,"time":1602402995666,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"DendronWorkspace","msg":"initialized"}
{"level":30,"time":1602402995667,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"_activate","installedGlobalVersion":"0.12.6","migratedGlobalVersion":"0.12.4","previousGlobalVersion":"0.12.3","previousWsVersion":"0.12.4","platform":"win32","extensions":[{"id":"dendron.dendron-paste-image","version":"1.0.4","active":false},{"id":"equinusocio.vsc-material-theme","version":"33.0.0","active":false},{"id":"dendron.dendron-markdown-shortcuts","version":"0.12.1","active":false},{"id":"dendron.dendron-markdown-preview-enhanced","version":"0.10.1","active":false},{"id":"dendron.dendron-markdown-links","version":"0.6.8","active":false},{"id":"dendron.dendron-markdown-notes","version":"0.0.17","active":false},{"id":"github.github-vscode-theme","version":"1.1.5","active":false}],"workspace":"a:\\Dokumente\\Notes\\vault"}
{"level":30,"time":1602402995667,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"_activate","msg":"wsActive","lspSupport":false}
{"level":30,"time":1602402995667,"pid":984,"hostname":"User-PC","name":"dendron","msg":"initialized"}
{"level":30,"time":1602402995668,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"getOrCreateEngine","msg":"create new engine"}
{"level":30,"time":1602402995669,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"activateWorkspace","stage":"dev","msg":"enter"}
{"level":30,"time":1602402995669,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"createWorkspaceWatcher"}
{"level":30,"time":1602402995670,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"createServerWatcher","portFile":"a:\\Dokumente\\Notes\\.dendron.port"}
{"level":30,"time":1602402995671,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"createSchemaWatcher","msg":"enter"}
{"level":30,"time":1602402995684,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"ReloadIndex.execute","msg":"enter"}
{"level":30,"time":1602402997804,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"ReloadIndex.execute","msg":"exit"}
{"level":30,"time":1602402997921,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"reloadWorkspace","msg":"post-reload-ws"}
{"level":30,"time":1602402997921,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"reloadWorkspace","msg":"exit"}
{"level":30,"time":1602402997922,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"postReloadWorkspace","msg":"preUpgrade: new wsVersion"}
{"level":30,"time":1602402997922,"pid":984,"hostname":"User-PC","name":"UpgradeSettingsCommand","ctx":"Upgrade:execute","opts":{"settingOpts":{"force":true}}}
{"level":30,"time":1602402998429,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"reloadWorkspace","msg":"gotConfig","config":{"folders":[{"path":"vault"}],"settings":{"editor.fontFamily":"'Cascadia Code SemiLight', 'Fira Code', Consolas, 'Courier New', monospace","editor.fontSize":16,"dendron.rootDir":".","editor.minimap.enabled":false,"files.autoSave":"onFocusChange","workbench.colorTheme":"Quiet Light","pasteImage.path":"${currentFileDir}/assets/images","pasteImage.prefix":"/","markdown-preview-enhanced.enableWikiLinkSyntax":true,"markdown-preview-enhanced.wikiLinkFileExtension":".md","vscodeMarkdownNotes.noteCompletionConvention":"noExtension","vscodeMarkdownNotes.slugifyCharacter":"NONE","editor.snippetSuggestions":"inline","editor.suggest.snippetsPreventQuickSuggestions":false,"editor.suggest.showSnippets":true,"editor.tabCompletion":"on","[markdown]":{"editor.quickSuggestions":true,"editor.wordBasedSuggestions":false,"editor.tabSize":2,"editor.suggest.showSnippets":true}},"extensions":{"recommendations":["dendron.dendron-paste-image","equinusocio.vsc-material-theme","dendron.dendron-markdown-shortcuts","dendron.dendron-markdown-preview-enhanced","dendron.dendron-markdown-links","dendron.dendron-markdown-notes","github.github-vscode-theme"],"unwantedRecommendations":["shd101wyy.markdown-preview-enhanced","kortina.vscode-markdown-notes","mushan.vscode-paste-image"]}}}
{"level":30,"time":1602407537348,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"show"}
{"level":30,"time":1602407537350,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"updatePickerBehavior","resp":[{"iconPathNormal":{"id":"find-selection"},"iconPathPressed":{"id":"menu-selection"},"type":"selectionExtract","pressed":true,"title":"Selection Extract"}]}
{"level":30,"time":1602407537350,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"updatePickerItems","querystring":"","opts":{"flavor":"schema"},"msg":"no qs"}
{"level":30,"time":1602407537351,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"updatePickerItems","querystring":"","opts":{"flavor":"schema"},"msg":"exit","profile":0}
{"level":30,"time":1602407537351,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"updatePickerItems","querystring":"","opts":{"flavor":"schema"},"msg":"no qs"}
{"level":30,"time":1602407537351,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"updatePickerItems","querystring":"","opts":{"flavor":"schema"},"msg":"exit","profile":0}
{"level":30,"time":1602407540085,"pid":984,"hostname":"User-PC","name":"dendron","ctx":"execute","opts":{}}
```

**Comment by onwijsbegeerte** [comment]
Running into the same issue on macOS. I’ll do some testing with removing some thing of the files from the vault later today.

**Comment by kevinslin** [comment]
this should now be fixed with 0.12.7. thanks for reporting!
